test/server.js

```diff
@@ -1,9 +1,10 @@
 const FtpSrv = require("ftp-srv");
+const path = require("path");
 
 // Using non-standard port
 const port = 2121;
-const homeDir =
-    require("os").homedir() + "/code/projects/ftp-deploy/test/remote";
+const homeDir = path.join(__dirname, "../test/remote");
```

Comment:
I made this change so that people on windows could run the tests - is there a reason for deleting it?

Reply:
Can we revert this change please. I explained why before